routes/routes.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-Route::get('/join/{recruitment_key}', ['as' => 'recruitment.join', 'uses' => '\Klsandbox\RecruitmentRoute\Http\Controllers\RecruitmentManagementController@getJoin']);
+Route::get('/join/{recruitment_key}/{role}', ['as' => 'recruitment.join', 'uses' => '\Klsandbox\RecruitmentRoute\Http\Controllers\RecruitmentManagementController@getJoin']);
```

Comment:
Dont do this - it make it look ugly. The role should come from the recruitment key itself